docs/How-to-add-new-gem-to-omsagent-bundle.md

```diff
@@ -0,0 +1,49 @@
+## How to add a new gem to omsagent install bundle
+
+In order to add a new Ruby gem into OMSAgent bundle when you have the new_gem_name.gem file on you local system you need to follow these steps:
+1. Update the private branch:
+* Clone Microsoft/Build-OMS-Agent-for-Linux github repo
+* Copy the new_gem_name.gem to Build-OMS-Agent-for-Linux//omsagent/source\ext\gems folder.
```

Comment:
Use consistent `/` formats that will actually work on Linux please. And rather than text like this, I prefer a concrete example. You can use your sample file, for example. Give REAL commands, that will help the reader to understand.